debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -255,17 +262,27 @@ public void dispatchTransactionMessage(SourceRecord record) throws InterruptedEx
     private final class StreamingChangeRecordReceiver implements ChangeRecordEmitter.Receiver {
 
         @Override
-        public void changeRecord(DataCollectionSchema dataCollectionSchema, Operation operation, Object key, Struct value, OffsetContext offsetContext)
+        public void changeRecord(DataCollectionSchema dataCollectionSchema,
```

Comment:
The formatting (line wrapping) changes mixed with actual code changes make it a bit harder to grasp than needed. Better to keep these things separate.